CONTRIBUTING.md

```diff
@@ -52,3 +52,8 @@ $> ./vendor/bin/phpcs --standard="vendor/aik099/coding-standard/CodingStandard"
 ```
 
 or by making your IDE ([instructions for PhpStorm](http://www.jetbrains.com/phpstorm/webhelp/using-php-code-sniffer-tool.html)) to check them automatically.
+
+As an aid to code formatting, a `.editorconfig` file is supplied.  If your IDE can support it, (instructions for 
+PHPStorm are [here](https://www.jetbrains.com/help/webstorm/configuring-code-style.html) and 
+[here](https://plugins.jetbrains.com/plugin/7294-editorconfig),) then please use it.  Alternatively, open the 
```

Comment:
It seems, that you've left trailing comma (`,`) before parenthesis (`)`), that needs to be removed.

Reply:
comma before closing parenthesis is correct grammar.

Reply:
or at least it is when I was at school. Opinions vary.

Reply:
I've double checked on https://www.grammarly.com/blog/comma-before-parenthesis-or-after/ and can't find any example where you can write `la-la,)` text VS `la-la),` text.